### ISSUE TITLE
Show all cards in a board

### ISSUE THREAD

**Issue by filipeabperes** [oryanm/TrelloWidget#27]
An interesting feature to add (at least for me it would be useful) might be to have the widget show all cards in a board, instead of just from a single list.  An intuitive way to fit this into the UI I believe would be to have an "All lists" option in the current lists menu when setting up the widget. 

Also, I checked and the Trello API does have a call for getting all cards in a board (https://developers.trello.com/advanced-reference/board#get-1-boards-board-id-cards), so this shouldnt be to hard on an addition.

**Comment by oryanm** [comment]
Is this something Trello itself lets you do in any way?

**Comment by filipeabperes** [comment]
I'm not sure exactly what you mean. If it's regarding the API, there's a call that Trello provides that allows you to do that (linked in the original post). If it's regarding their interface, I suppose they do, since the standard view is to open a board and see all cards there (grouped by list of course, but you can still see all cards for that board).

**Comment by oryanm** [comment]
I was regarding the UI because I don't think it would be very intuitive to have an entire board in one list. I understand your request but since Trello doesn't have this feature I don't think the widget should either. (You could just have multiple widgets to represent all of the board. This works best using a tablet)

**Comment by filipeabperes** [comment]
Sure. The original widget has similar functionality, in that you can see all tasks assigned to you despite being on same or different cards. This would just be a generalization of that. This seems useful to me because, having many widgets in a phone is impractical, and you might just want to be able to have a glance at everything that is on a certain board, while using the full Trello interface for a more detailed breakdown of the separate lists. But it's just a suggestion, so feel free to discard it hahah

**Comment by oryanm** [comment]
If I were to do something like this it would be more of a "create your own query" feature that is more general than just "all cards in a board". But no promises :) I'll leave this open for now.

**Comment by filipeabperes** [comment]
yeah, that sounds pretty useful as well. no pressure hahah and thanks for the app in general, its pretty useful
